common/sgx/internal.edl

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** internal.edl:
+**
+**     This file declares internal ECALLs/OCALLs used by liboehost/liboecore.
+**
+**==============================================================================
+*/
+
+enclave {
+
+    trusted {
+
+        /* This function returns oe_internal_ping_ocall(value). */
+        public int oe_internal_ping_ecall(int value);
```

Comment:
It is not clear what _ping_ signifies. Consider using a more meaningful name.

Reply:
It's a placeholder since EDL requires you define at least one ECALL. This will be removed in Part II of this PR, which adds the actual calls.

Reply:
Ah ok. I genuinely though this was part of some kind of ping-pong scheme for initializing enclaves.

Reply:
I added a comment that this will be removed.